src/components/user/profile/header.tsx

```diff
@@ -35,36 +35,36 @@ const Header = ({
       <div className="flex max-h-full w-full max-w-full flex-col gap-1">
         <p className="text-3xl font-bold">{name}</p>
 
-        <div className="flex items-center gap-2">
-          <Mail className="h-5 w-5 text-starlight-gray" />
+        <div className="flex items-center">
           <Link
             href={`mailto:${email}`}
             className="text-starlight-gray"
             target="_blank"
           >
-            {email}
+            <Mail className="inline-block h-5 w-5 text-starlight-gray" />
+            <span className="ml-2">{email}</span>
```

Comment:
used margin left in a separate span because icons weren't aligning but, not sure if this is the best way to do it.